gemini/evaluation/metrics_based_gemini_evaluation.ipynb

```diff
@@ -0,0 +1,646 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        response = generation_model.generate_content(contents=prompt).text
</pre><p>Did you intend to not include the generation_config?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/482/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>